### ISSUE TITLE
Listed components should be clickable anywhere on the area

### ISSUE THREAD

**Issue by Jerryescandon** [GCTC-NTGC/gc-digital-talent#3704]
## Description

The areaa between and after the label on a list item (radio,checkboxes) should be clickable, as opposed to 'only' the text label or the box/circl.e

## Screenshot: Prototype
![image](https://user-images.githubusercontent.com/16977254/185472189-684dd287-b00d-4926-a61d-e05c0eda9c31.png)

## Acceptance Criteria
- [ ] ...

   [image]: https://user-images.githubusercontent.com/305339/166570935-6425a615-82c7-4b2f-9840-5c7d698fa2ab.png
   [image-link]: https://example.com
   [figma]: https://go.talent.c4nada.ca/figma
   [sitemap]: https://go.talent.c4nada.ca/sitemap
